api/api-doc/column_family.json

```diff
@@ -663,8 +711,8 @@
                "parameters":[
                   {
                      "name":"name",
-                     "description":"The column family name in keyspace:name format",
-                     "required":true,
+                     "description":"The column family name in keyspace:name format. Omitting the name (i.e. 'keyspace:') will run the query on the whole keyspace",
```

Comment:
This is not correct,  the path parameter should be required, if the name is omitted, you would fall to your new implementation that does not have a query name parameter

Reply:
Thanks for pointing that out, forgot to change it back to required